app/src/main/java/com/example/ioana/travel_journal/HomeFragment.java

```diff
@@ -215,4 +213,6 @@ public void onComplete(@NonNull Task<QuerySnapshot> task) {
                 });
         return trips;
     }
+
```

Comment:
Can someone try to implement in the same manner onLongPressListener (which will redirect us to TripDetailsActivity)?